src/models/Project/Project.ts

```diff
@@ -1,9 +1,10 @@
 import { Address } from '../Address/Address';
 
-export interface Project {
+export interface ProjectData {
     id?: string;
     name?: string;
     description?: string;
+    clientId?: string | null;
```

Comment:
You can remove the `null` here and just go with the `?`

Reply:
ok